homeassistant/components/knx/__init__.py

```diff
@@ -71,62 +72,72 @@
 SERVICE_KNX_ATTR_ADDRESS = "address"
 SERVICE_KNX_ATTR_PAYLOAD = "payload"
 SERVICE_KNX_ATTR_TYPE = "type"
+SERVICE_KNX_ATTR_REMOVE = "remove"
```

Comment:
We don't need to include `ATTR_` in service name constants. That's only used for service data keys and state attribute keys.
```suggestion
SERVICE_KNX_REMOVE = "remove"
```